web/src/layout/main.js

```diff
@@ -57,16 +57,17 @@ const Main = ({ brightModeEnabled, electricityMixMode, isLeftPanelCollapsed }) =
             </a>
           </div>
           <Legend />
-          <Toggle
-            className="production-consumption-toggle"
```

Comment:
Good idea with the wrappers! I think it's ok to remove the `className` prop from the`Toggle` component if we're going to be using wrappers instead.